packages/application-generic/src/utils/filter-processing-details.ts

```diff
@@ -1,10 +1,11 @@
 import { StepFilter, SubscriberEntity } from '@novu/dal';
-import { ICondition, ITriggerPayload } from '@novu/shared';
+import { ICondition, ITenantDefine, ITriggerPayload } from '@novu/shared';
 
 export interface IFilterVariables {
   payload?: ITriggerPayload;
   subscriber?: SubscriberEntity;
   webhook?: Record<string, unknown>;
+  tenant?: ITenantDefine;
```

Comment:
this should be TenantEntity IMO. as we have a filter with the thenat name, and in the ITenantDefine we only force to have an identifier. 

Should be similar to the subscriber